.buildconfig.yml

```diff
@@ -331,7 +331,7 @@ projects:
   tooling-detekt:
     path: components/tooling/detekt
     description: 'Custom Detekt rules for internal use.'
-    publish: true
+    publish: false
```

Comment:
Will not be needed once https://github.com/mozilla-mobile/android-components/pull/4645 is merged.

Reply:
👍 Rebased